services/feedback-request.js

```diff
@@ -8,7 +8,7 @@
 const { ActionRowBuilder, EmbedBuilder, SelectMenuBuilder, ModalBuilder, TextInputBuilder, TextInputStyle, ButtonBuilder, ButtonStyle } = require('discord.js');
 const { feedbackChannelId, apiUrlPrefix } = require('../config.json');
 const updateMessage = require('./message-update');  // Contains useful methods to update the messages shown to users
-const jwtVerify = require('./jwt-verification'); // Used to update the JWT
+const jwtVerify = require('../../el_project_api/jwt-verification'); // Used to update the JWT
```

Comment:
We just were wandering why this path was changed. It probably was better the first way around since we do not know for sure the name of the project folder as appearing in the server of the hosting provider. Except if there is a specific reason for which u had to change it which I'd be happy to discuss

Reply:
Wasn't intentional, we can keep it how it was